.claude-skills/registry/SKILL_REGISTRY.md

```
@@ -200,9 +200,10 @@ Products:
 - gym-training-expert
 - health-nutrition-expert
 
-### Project Skills (2)
+### Project Skills (3)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)